source/quickstart/osx.rst

```diff
@@ -6,31 +6,213 @@ Quickstart for Mac OSX
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
+         $ mc alias set myminio http://192.0.2.10:9000 minioadmin minioadmin
+
+      Documentation: https://docs.min.io
+
+      WARNING: Detected default credentials 'minioadmin:minioadmin', we recommend that you change these values with 'MINIO_ROOT_USER' and 'MINIO_ROOT_PASSWORD' environment variables.
+
+#. Access the :ref:`minio-console` by going to a browser (such as Safari) and entering the Console address specified in the :mc:`minio server` command's output.
+   For example, http://127.0.0.1:9001.
+
+   Log in to the Console with the ``RootUser`` and ``RootPass`` user credentials displayed in the output.
+   These default to ``minioadmin | minioadmin``.
+
+   .. image:: /images/minio-console-dashboard.png
+      :width: 600px
+      :alt: MinIO Console Dashboard displaying Monitoring Data
+      :align: center
+
+   You can use the MinIO Console for general administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration. 
+   Each MinIO server includes its own embedded MinIO Console.
+
+   From the console, you can create and manage buckets, add objects, and many other tasks.
+   For more information, see the :ref:`minio-console` documentation.
+
+
+Optional Steps
+--------------
+
+Install the MinIO Client
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`MinIO Client <minio-client>` allows you to work with your MinIO volume from the commandline.
+
+.. tab-set::
+
+   .. tab-item:: Homebrew
+
+      Run the following commands to install the latest stable MinIO Client package using `Homebrew <https://brew.sh>`_.
+
+      .. code-block:: shell
+         :class: copyable
+
+         brew install minio/stable/mc
+
+   .. tab-item:: Binary
+
+      Download the standalone MinIO server for MacOS and make it executable.
+            
+      .. code-block:: shell
+         :class: copyable
+
+         wget https://dl.min.io/server/minio/release/darwin-amd64/minio
+         chmod +x minio
+
+Add TLS Certificates
+~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports enabling :ref:`minio-tls` 1.2+ automatically upon detecting an x.509 private key (``private.key``) and public certificate (``public.crt``) in the MinIO ``certs`` directory:
+
+- For MacOS: ``${HOME}/.minio/certs``
+
+You can override the certificate directory using the :mc:`minio-server --certs-dir` commandline argument.
+
+Run the MinIO Server with Non-Default Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Issue the following command to start the :mc:`minio server` with non-default credentials. 
+The table following this command breaks down each portion of the command:
+
+.. code-block:: shell
+   :class: copyable
+
+   export MINIO_ROOT_USER=minio-admin
+   export MINIO_ROOT_PASSWORD=minio-secret-key-CHANGE-ME
+   #export MINIO_SERVER_URL=https://minio.example.net
```

Comment:
We can drop this, it's only necessary if TLS is enabled and DNS is actually configured.